functions/char_count.py

```diff
@@ -0,0 +1,39 @@
+def count_char(string,char):
+	""" Find the number of occurrence of the passed character agument
+		from the passed string argument. Function returns the count 
+		of the character in the given string, if the characters is 
+		not found returns -1.
+
+        Args:
+            string: Argument 1 sould be a string from which need to find
+					the number of occurrence of character.
+            input2: Argument 2 should be a character.
+
+        Returns:
+            Returns the number of occurrence of the character in 
+			the given string. If the character is not found in the string
+			returns -1.
+    """
+
+	value = string.count(char)
+	""" count() method to find the count of occurrence of char
```

Comment:
Remove this inline comment. It will affect the readability of the code. Not all the place requires inline comments.